crates/driver/src/domain/competition/auction.rs

```diff
@@ -130,10 +132,14 @@ impl Auction {
 #[derive(Clone)]
 pub struct AuctionProcessor(Arc<Mutex<Inner>>);
 
+type OrderComparator =
```

Comment:
worth defining a trait for it, or unless document what is each field (some are obvious but for example `eth::H160` isn't).

Reply:
True. Defined a trait.